object/permission.go

```diff
@@ -57,6 +57,40 @@ type PermissionRule struct {
 	Id    string `xorm:"varchar(100) index not null default ''" json:"id"`
 }
 
+const (
```

Comment:
Now, `permission` default uses table named "permission_rule" as adapter.  `permission` cannot select other adapter. 
The field `builtInAdapter` is reserved to distinguish the other adapter in the futhur, if casdoor implement` permisson` can select exist adapter flexible.